contracts/pool/ConstantProductPool.sol

```diff
@@ -20,12 +19,14 @@ contract ConstantProductPool is IPool, TridentERC20 {
 
     uint256 internal constant MINIMUM_LIQUIDITY = 1000;
 
-    uint8 internal constant PRECISION = 112;
     uint256 internal constant MAX_FEE = 10000; // @dev 100%.
     uint256 internal constant MAX_FEE_SQUARE = 100000000;
     uint256 internal constant E18 = uint256(10)**18;
     uint256 public immutable swapFee;
     uint256 internal immutable MAX_FEE_MINUS_SWAP_FEE;
+    uint8 internal constant PRECISION = 112;
+    
+    uint8 internal unlocked;
```

Comment:
Precision is inlined in the code and not part of the storage. Meaning we can't pack it together with something else. Unlocked should stay in its default 256 slot so the calculations don't have to do the conversion